docs/source/get-started/quickstart.md

```diff
@@ -0,0 +1,274 @@
+# Quickstart
+
+Now that you have a local deployment of Lumigator, you can start using it. In this quickstart guide,
+we will show you how to upload a dataset and create a simple evaluation job. Finally, we'll show you
+how to retrieve the results of the evaluation job.
+
+## Upload a Dataset
+
+The Lumigator backend provides an API endpoint for uploading datasets and running evaluation jobs.
+To view the available endpoints, navigate to the API documentation page at
+[`http://localhost:8000/docs`](http://localhost:8000/docs).
+
+There are a few ways to interact with the API; you can always test the endpoints via the OpenAPI
+documentation page, but here we will show you how to send requests using cURL and the Lumigator
+Python SDK.
+
+To upload a dataset, you need to send a POST request to the `/datasets` endpoint. The request should
+include the dataset file. Here is an example:
+
+::::{tab-set}
+
+:::{tab-item} cURL
+:sync: tab1
+```console
+user@host:~/lumigator$ curl -s http://localhost:8000/api/v1/datasets/ \
+  -H 'Accept: application/json' \
+  -H 'Content-Type: multipart/form-data' \
+  -F 'dataset=@'"path/to/dataset.csv"';type=text/csv' \
+  -F 'format=experiment' | jq
+{
+  "id": "dd15bbaa-8d6f-44ae-a995-b3b78f4ea6fb",
+  "filename": "dataset.csv",
+  "format": "experiment",
+  "size": 180528,
+  "ground_truth": true,
+  "created_at": "2024-10-30T12:10:18"
+}
+```
+:::
+
+:::{tab-item} Python SDK
+:sync: tab2
+```python
+from lumigator_sdk.lumigator import LumigatorClient
+from schemas.datasets import DatasetFormat
+
+dataset_path = 'path/to/dataset.csv'
+lm_client = LumigatorClient('localhost:8000')
+
+response = lm_client.datasets.create_dataset(
+    open(dataset_path, 'rb'),
+    DatasetFormat.JOB
+)
+```
+:::
+
+::::
+
+```{note}
+The dataset file should be in CSV format and contain a header row with the following columns:
```

Comment:
this doesn't need to have `ground_truth`, it's an optional field, we could note that and point people to the example dataset (https://github.com/mozilla-ai/lumigator/blob/0bef1965c5180f39832e2932b59ef797b0853ff4/lumigator/python/mzai/sample_data/dialogsum_exc.csv#L4)